src/app/enrollment/pages/tools-and-forms/tools-and-forms.component.html

```diff
@@ -63,11 +63,11 @@ <h5>PRODUCT INFORMATION</h5>
 <div class="pdf-viewer" *ngIf="pdfSrc">
   <div class="action-panel">
     <app-custom-button
-      btnClass="x-small secondary"
```

Comment:
the purpose behind using small x-small was so that we dont have to create custom style for each button. revert this